pandas/tests/extension/test_categorical.py

```diff
@@ -193,6 +193,11 @@ def test_searchsorted(self, data_for_sorting):
         if not data_for_sorting.ordered:
             raise pytest.skip(reason="searchsorted requires ordered data.")
 
+    def test_argsort_nan_last(self, data_missing_for_sorting):
```

Comment:
why is this being skipped?

Reply:
The `NA-first` behaviour of `Categorical` will be changed to `NA-last` in the next PR. So the original `_values_for_argsort` is placed here temporarily.

The corresponding test for `NA-first` behaviour of `Categorical` will also be changed.